quirks/websites-with-shared-credential-backends.json

```diff
@@ -109,7 +109,8 @@
     [
         "disneystore.com",
         "shopdisney.com",
-        "go.com"
+        "go.com",
+        "disneyplus.com"
```

Comment:
Can you provide URLs at all of these domains where people can log in with the same accounts? In your commit message/pull request.

Reply:
Added to PR, additionally removed disneystore.com as that now just redirects straight to shopdisney.com